pkg/test/framework/api/descriptors/descriptors.go

```diff
@@ -41,7 +41,6 @@ var (
 		Requires: []component.Requirement{
 			&ids.Mixer,
 			&ids.Environment,
-			&ids.Galley,
```

Comment:
It looks like this shouldn't be part of this pack?

Reply:
Fixed